docs/semgrep-assistant/analyze.md

```diff
@@ -1,16 +1,16 @@
 ---
 slug: analyze
-title: Analyze findings
+title: Analyze Code findings
```

Comment:
This is the Assistant > Analyze document. Because we now have more differentiated functions for SSC, I just added some  text to differentiate that this is for Semgrep Code. Happy to have feedback @khorne3

Reply:
I think this is fine for now!